automated install/basic-install.sh

```diff
@@ -2149,7 +2134,7 @@ update_dialogs() {
         strAdd="You will be updated to the latest version."
     fi
     opt2a="Reconfigure"
-    opt2b="This will reset your Pi-hole and allow you to enter new settings."
+    opt2b="This resets settings and allows to reselect them."
 
```

Comment:
How about `Resets Pi-hole and allows re-selecting settings.' ?

Reply:
Jep, sounds better.